contracts/metadata/metadata.cairo

```diff
@@ -0,0 +1,691 @@
+        const wonder_28 = 'The Fading Yew';
+        const wonder_29 = 'The Origin Oasis';
+        const wonder_30 = 'The Sanctified Fjord';
+        const wonder_31 = 'The Pale Pillar';
+        const wonder_32 = 'Sanctum Of The Oracle';
+        const wonder_33 = 'The Ethereal Isle';
+        const wonder_34 = 'The Omen Graves';
+        const wonder_35 = 'The Pale Vertex';
+        const wonder_36 = 'The Glowing Pinnacle';
+        const wonder_37 = 'The Azure Lake';
+        const wonder_38 = 'The Argent Catacombs';
+        const wonder_39 = 'The Dark Mountain';
+        const wonder_40 = 'Sky Mast';
+        const wonder_41 = 'Infinity Spire';
+        const wonder_42 = 'The Exalted Basin';
+        const wonder_43 = 'The Ancestral Trees';
+        const wonder_44 = 'The Perpetual Fjord';
+        const wonder_45 = 'The Ancient Lagoon';
+        const wonder_46 = 'The Pearl River';
+        const wonder_47 = 'The Cerulean Reliquary';
+        const wonder_48 = 'Altar Of Divine Will';
+        const wonder_49 = 'Pagoda Of Fortune';
+        const wonder_50 = 'The Oracle Pool';
+    }
+}
+
+namespace Uri {
+    // @notice build uri array from stored realm data
+    // @implicit syscall_ptr
+    // @implicit bitwise_ptr
+    // @implicit range_check_ptr
+    // @param realm_id: id of the realm
+    // @param realm_name: encoded string of the realm name
+    // @param realm_data: unpacked data for realm
+    // @param realm_type: type of realm (Realm or S_Realm)
+    func build{
+        syscall_ptr: felt*,
+        pedersen_ptr: HashBuiltin*,
+        range_check_ptr,
+        bitwise_ptr: BitwiseBuiltin*,
+    }(realm_id: Uint256, realm_name: felt, realm_data: RealmData, realm_type: felt) -> (
+        encoded_len: felt, encoded: felt*
+    ) {
+        alloc_locals;
+
+        // pre-defined for reusability
+        let left_bracket = Utils.Symbols.LeftBracket;
+        let right_bracket = Utils.Symbols.RightBracket;
+        let inverted_commas = Utils.Symbols.InvertedCommas;
+        let comma = Utils.Symbols.Comma;
+
+        let data_format = 'data:application/json,';
+
+        // keys
+        let description_key = '"description":';
+        let name_key = '"name":';
+        let image_key = '"image":';
+        let attributes_key = '"attributes":';
+
+        let left_square_bracket = 91;
+        let right_square_bracket = 93;
+
+        // get value of description
+        let description_value = '"realms"';
+
+        // realm image url values
+        let realm_image_url_1 = 'https://';
+        let realm_image_url_2 = 'd23fdhqc1jb9no';
+        let realm_image_url_3 = '.cloudfront.net';
+        let realm_image_url_4 = '/_Realms/';
+
+        // s realm image url values
+        let s_realm_image_url_1 = 'https://';
+        let s_realm_image_url_2 = 'realms-asse';
+        let s_realm_image_url_3 = 'ts.s3.eu-west-';
+        let s_realm_image_url_4 = '3.amazonaws.com';
+        let s_realm_image_url_5 = '/renders/';
```

Comment:
Can you merge both `realm_image_url_*` and `s_realm_image_url_*` into two split strings? Slightly more efficient and no need for the hack on line 215.